osgeo_importer/forms.py

```diff
@@ -81,7 +90,19 @@ def clean(self):
                     'Inspector could not read file {} or file is empty'.format(cleaned_file_path)
                 )
                 continue
+            upload_size += os.path.getsize(cleaned_file_path)
             inspected_files.append(cleaned_file)
 
         cleaned_data['file'] = inspected_files
+        # Get total file size
+        cleaned_data['upload_size'] = upload_size
+        if USER_UPLOAD_QUOTA is not None:
+            # Get the total size of all data uploaded by this user
+            user_filesize = UploadedData.objects.filter(user=self.request.user).aggregate(s=Sum('size'))['s']
+            if user_filesize is None:
+                user_filesize = 0
+            if user_filesize + upload_size > USER_UPLOAD_QUOTA:
+                # remove temp directory used for processing upload if quota exceeded
+                shutil.rmtree(outputdir)
+                self.add_error('file','User Quota Exceeded. Quota: %s Used: %s Adding: %s'%(USER_UPLOAD_QUOTA, user_filesize, upload_size))
```

Comment:
Adding the units kb, mb, etc would be helpful for general users to quickly ascertain the quota. e.g User Quota Exceeded. Quota: 100kb Used: 100kb Adding: 100MB.